pkg/reconciler/intevents/topic/topic.go

```diff
@@ -156,8 +160,16 @@ func (r *Reconciler) reconcileTopic(ctx context.Context, topic *v1.Topic) error
 			logging.FromContext(ctx).Desugar().Error("Topic does not exist and the topic policy doesn't allow creation")
 			return fmt.Errorf("Topic %q does not exist and the topic policy doesn't allow creation", topic.Spec.Topic)
 		} else {
+			topicConfig := &pubsub.TopicConfig{}
+			if r.dataresidencyStore != nil {
+				if dataresidencyCfg := r.dataresidencyStore.Load(); dataresidencyCfg != nil {
+					if dataresidencyCfg.DataResidencyDefaults.ComputeAllowedPersistenceRegions(topicConfig) {
+						r.Logger.Infow("Updated Topic Config", zap.Any("topicConfig", *topicConfig))
```

Comment:
Same comment as above: this should be Debug with a more specific message.

Reply:
Fixed as suggested.